README.md

```diff
@@ -42,6 +42,28 @@ virtualenv ./venv
 source ./venv/bin/activate
 ```
 
+## Install dependencies
+
+You will need `libffi`, `libxml2`.
+
+On OS X, make sure you have XCode CLI installed:
+
+```
+xcode-select --install
+```
+
+Then install the formulae:
+
+```
+brew install libffi libxml2 cairo pango
+```
+
+On Debian-based Linux distros (e.g. Ubuntu):
+
+```
+sudo apt-get install libffi-dev libxml2-dev libcairo2-dev pango-graphite
```

Comment:
The dev version of the ffi package might be needed, but I'm pretty sure the plain versions are enough for the others.  I'm not using a Debian-based system so I can't confirm.

Actually, I don't think any of us non-Mac users have had a problem with this yet.